src/nvim/eval.c

```diff
@@ -14266,7 +14266,8 @@ static void f_serverstart(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 
   int result = server_start((char *) rettv->vval.v_string);
   if (result != 0) {
-    EMSG2("Failed to start server: %s", uv_strerror(result));
+    EMSG2("Failed to start server: %s",
+          result > 0 ? "Unknonwn system error" : uv_strerror(result));
```

Comment:
> `result` is here always `1`

`:call serverstart('localhost:12345')` twice throws the exception: `Failed to start server: invalid argument` (`UV_EINVAL`) for me.